### DISCUSSION TITLE
Color Management with  "Little CMS" Open Source

### DISCUSSION THREAD

**Discussion by donatopace** [d2phap/ImageGlass#1417] [General]
This is the official description of Little CMS
Can you integrate it under Imageglass?
Thanks.

This is an OPEN SOURCE small-footprint color management engine, with special focus on accuracy and performance. It uses the International Color Consortium standard (ICC), which is the modern standard when regarding to color management. The ICC specification is widely used and is referred to in many International and other de-facto standards. It was approved as an International Standard, ISO 15076-1, in 2005.

https://littlecms.com/color-engine/

**Comment by d2phap** [comment]
ImageGlass uses Magick.NET to read and process image. Using another library will slow down the loading speed because the image data will need to be converted, transferred into other types

**Reply by donatopace** [reply]
Hello,
i found in the repository that "Magick" use LCMS (Little CMS) library natively,
Can you please improve color management?
Thanks.
Example of problem below.
https://github.com/d2phap/ImageGlass/issues/1577
